acceptance/tests/resource/user/should_modify_uid.rb

```diff
@@ -1,6 +1,5 @@
 test_name "should modify the uid of an user OS X < 10.14" do
   confine :to, :platform => /osx/
-  confine :except, :platform => /(osx-10.1[4-9]|osx-11-)/
```

Comment:
This contradicts the `osx_10.4_should_fail_when_modify_uid.rb` test and will not pass since we are unable to change the home and uid on OS X greater than 10.14.

Changes to the other tests should work since we no longer test older versions of macOS (that supported changing the user's home and uid).